.github/workflows/stale.yaml

```diff
@@ -0,0 +1,36 @@
+# This workflow warns and then closes issues and PRs that have had no 
+# activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see: https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '0 4 * * *'
```

Comment:
I think we can increase how often this job is run. It would make me kind of sad updating a issue/pr and for the entire day it keeps saying stale.

Reply:
That is true, but isn't that best addressed by also running the workflow when there are comments, commits, et c?

Reply:
Yeah we can do that as well, but if we are using that I guess we can disable the cronjob completely. If we trigger the job on any comment it will check all the issue not just the issue that we commented on.